Epoch 1/10:   0%|                                                                                                                                         | 0/1353 [00:00<?, ?it/s]
Traceback (most recent call last):
  File "/home/ali/GScan/GScanDeepLearning/MuonClassifier/train.py", line 64, in <module>
    for i, data in enumerate( tqdm(train_dataloader, desc=f'Epoch {epoch + 1}/{n_epochs}')):
  File "/home/ali/.miniforge3/envs/pip_only/lib/python3.8/site-packages/tqdm/std.py", line 1182, in __iter__
    for obj in iterable:
  File "/home/ali/.miniforge3/envs/pip_only/lib/python3.8/site-packages/torch/utils/data/dataloader.py", line 630, in __next__
    data = self._next_data()
  File "/home/ali/.miniforge3/envs/pip_only/lib/python3.8/site-packages/torch/utils/data/dataloader.py", line 1345, in _next_data
    return self._process_data(data)
  File "/home/ali/.miniforge3/envs/pip_only/lib/python3.8/site-packages/torch/utils/data/dataloader.py", line 1371, in _process_data
    data.reraise()
  File "/home/ali/.miniforge3/envs/pip_only/lib/python3.8/site-packages/torch/_utils.py", line 694, in reraise
    raise exception
FileNotFoundError: Caught FileNotFoundError in DataLoader worker process 0.
Original Traceback (most recent call last):
  File "/home/ali/.miniforge3/envs/pip_only/lib/python3.8/site-packages/torch/utils/data/_utils/worker.py", line 308, in _worker_loop
    data = fetcher.fetch(index)
  File "/home/ali/.miniforge3/envs/pip_only/lib/python3.8/site-packages/torch/utils/data/_utils/fetch.py", line 51, in fetch
    data = [self.dataset[idx] for idx in possibly_batched_index]
  File "/home/ali/.miniforge3/envs/pip_only/lib/python3.8/site-packages/torch/utils/data/_utils/fetch.py", line 51, in <listcomp>
    data = [self.dataset[idx] for idx in possibly_batched_index]
  File "/home/ali/GScan/GScanDeepLearning/MuonClassifier/dataloader.py", line 52, in __getitem__
    front_image= self.transforms(Image.open(input_front))
  File "/home/ali/.miniforge3/envs/pip_only/lib/python3.8/site-packages/PIL/Image.py", line 3243, in open
    fp = builtins.open(filename, "rb")
FileNotFoundError: [Errno 2] No such file or directory: '/home/ali/GScan/GScanDeepLearning/MuonClassifier/data_classification_loose/train/1382_yes/input_front.png'
